CondTools/Geometry/plugins/PGeometricDetBuilder.cc

```diff
@@ -189,7 +189,7 @@ void PGeometricDetBuilder::putOne(const GeometricDet* gd, PGeometricDet* pgd, in
     item._params9 = 0;
     item._params10 = 0;
   }
-  item._geographicalID = gd->geographicalID();
+  item._geographicalID = gd->geographicalId();
```

Comment:
`geographicalID` is actually correct, but it is not worth changing back.

Reply:
They serve the same purpose https://cmssdt.cern.ch/lxr/source/Geometry/TrackerNumberingBuilder/interface/GeometricDet.h#0151